forge-game/src/main/java/forge/game/spellability/SpellAbilityStackInstance.java

```diff
@@ -172,33 +112,12 @@ public int getId() {
     //      Perhaps lets move the refresh logic to a separate function called only when necessary
     public final SpellAbility getSpellAbility(boolean refresh) {
         if (refresh) {
-            ability.setTargets(tc);
             ability.setActivatingPlayer(activatingPlayer);
```

Comment:
is that the only thing getSpellAbility(true) does now? reset the Activating Player?

Reply:
yep! In theory it can be ripped out completely now, I will check